core/services/job/job_orm_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"github.com/smartcontractkit/chainlink/v2/core/services/relay"
```

Comment:
hmmm the linter should complain about this import being mixed in with the std lib :confused: 
cc @chainchad

Reply:
Maybe it ignores test files?

Reply:
Maybe I am forgetting a test file exception? If so, we may want to reconsider that.